src/layouts/_proxied-dot-io/vault/index.tsx

```diff
@@ -11,11 +11,14 @@ import Footer from 'components/_proxied-dot-io/vault/footer-with-props'
 import ProductSubnav from 'components/_proxied-dot-io/vault/subnav'
 import productData from 'data/vault.json'
 import query from './query.graphql'
+import { trackGoal as trackFathomGoal } from 'fathom-client'
```

Comment:
nit can you move this to line 2 so that we keep our external deps above our internal ones?